### DISCUSSION TITLE
In Postgresql, add a setting to auto-capitalize keywords like 'SELECT' and 'GROUP BY'

### DISCUSSION THREAD

**Discussion by DeflateAwning** [dbeaver/dbeaver#21238] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I'm used to other SQL versions, where these keywords are caps. I want it to be like this in Postgres, even if it's not that "standard". 

**Describe the solution you'd like**
I want a checkbox setting related to the SQL Text Editor called "Capitalize SQL Keywords?" or similar. 

**Additional context**
In all recent versions, when writing code while connected to a PostgreSQL DB, keywords like SELECT and GROUP BY are converted to lowercase.

**Comment by E1izabeth** [comment, marked as answer]
You can change the "Insert case" connection setting for autocompletion to "Upper case"
![image](https://github.com/dbeaver/dbeaver/assets/28875055/2dca2d46-d383-442e-bc87-58771d8379d1)

If you use formatter, you can change it there too
![image](https://github.com/dbeaver/dbeaver/assets/28875055/b7ccadec-d1c4-407b-86cc-6f22160a50a8)

**Comment by DeflateAwning** [comment]
Thank you!

**Comment by DeflateAwning** [comment]
Is there a way to keep table names as lower case, but have all built-in keywords uppercase? We want table names and column names lower case still
